org.civicrm.bootstrapcivihr/scss/directives/horizontal-scroll-shadows.scss

```diff
@@ -0,0 +1,40 @@
+.horizontal-scroll-shadows-master {
+  overflow: hidden; // this ensures the shadows are hidden behind
+  position: relative;
+}
+
+.horizontal-scroll-shadows-wrapper {
+  overflow-x: auto;
+
+  &::before,
+  &::after {
+    box-shadow: 0 0 10px 1px rgba(48, 40, 40, 0.5);
+    content: '';
+    display: block;
+    height: calc(100% - 10px); // if height is 100% then the shadow will be cut off
+    opacity: 0; // by default the shadows are invisible
```

Comment:
Wouldn't `opacity: 0` stop the user from clicking the content under the shadow even if it's not visible?

Reply:
The actual element is not even in the viewport, it is hidden completely from user. The shadow does not overlay the content in terms of click propagation. I considered this and tested, it does allow to click through.